core/src/main/java/jenkins/model/ModelObjectWithContextMenu.java

```diff
@@ -128,6 +128,19 @@ public ContextMenu add(String url, String icon, String text, boolean post, boole
             return this;
         }
 
+        /** @since 2.87 */
+        public ContextMenu add(String url, String icon, String text, boolean post, boolean requiresConfirmation,
```

Comment:
The other `add` methods probably should start calling this one to limit the code duplication.